server/src/commandLine.mjs

```diff
@@ -89,6 +90,13 @@ const developerToolName = parsed.developerToolName || config.app.developerToolNa
 const proxy = parsed.proxy;
 const conduit = parsed.conduit;
 
```

Comment:
**Note:** test() is a JS inbuilt RegExp method to test for a match in a string. Adding a note here to avoid any confusions
Documentation for reference- https://www.w3schools.com/jsref/jsref_regexp_test.asp